sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -182,8 +182,11 @@ export function record(
         (runtime === "browser" && isBrowser()) ||
         !runtime
       ) {
-        // record mode - recorder is stopped
-        if (isRecordMode()) recorder.stop();
+        // Since Mocha 7.0.0, Mocha behaves as follows:
+        // "When conditionally skipping in a it test, related afterEach hooks are now executed"
+        // Source: https://github.com/mochajs/mocha/blob/master/CHANGELOG.md#700--2020-01-05
+        // Therefore, now the recorder.stop() calls in the afterEach() will always be executed.
+        // if (isRecordMode()) recorder.stop();
```

Comment:
We probably would never have to look at this since mocha's behavior is not going to change.
So, I'd recommend removing the comments, description in the PR should be enough.

Reply:
sounds good! I'll polish this up in a bit